Python/Module1_GettingStartedWithPython/GettingStartedWithPython.md

```diff
@@ -144,6 +144,39 @@ As such, Python is a language that is conducive to rapidly prototyping and testi
 We will be relying heavily on Python and a library for doing optimized numerical computations, called NumPy, throughout this course.
 <!-- #endregion -->
 
+<!-- #region -->
+## Keeping Up To Date
+
+New versions of Python come out periodically, bringing new features and fixes. 
+The most recent version of Python is Python 3.8.4.
+Unfortunately, it takes time for packages to support new versions of Python.
+As a result, many packages don't support Python 3.8, and will require you to install Python 3.7 or, in some cases, 3.6.
+
+All Python versions numbers use the A.B.C format. 
```

Comment:
```suggestion
All Python version numbers use the A.B.C format. 
```

Further, they conform to [semantic versioning](https://semver.org/) - do we want to include a link?

Reply:
I think a link would be good